docs/contrib/tutes/setup-linux.rst

```diff
@@ -27,11 +28,17 @@ You're good to go.
 Opinionated Recommendations
 ===========================
 
-* Install Python with `pyenv`_
+* Install Python with pyenv
+  * Install dependencies: ```sudo apt-get install -y make build-essential libssl-dev zlib1g-dev libbz2-dev \
```

Comment:
Here maybe best to just link to the prerequisites document? https://github.com/pyenv/pyenv/wiki/common-build-problems#prerequisites

Reply:
Also I believe this line is what the tests are bawking about:

https://github.com/pymedphys/pymedphys/pull/558/checks?check_run_id=271464861#step:10:197

I don't think triple ``` is valid reStructured text. But if this line goes and is replaced by the requirements link that should fix that issue.